dev-packages/env-tests/cloudflare-astro/astro.config.mjs

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'astro/config';
+import cloudflare from '@astrojs/cloudflare'
+import sentry from '@sentry/astro'
+
+// https://astro.build/config
+export default defineConfig({
+  output: 'hybrid',
+  adapter: cloudflare({
+    imageService: 'passthrough',
+  }),
+  integrations: [
+    sentry({
+      dsn: process.env.E2E_TEST_DSN,
+      autoInstrumentation: {
+        requestHandler: true,
+      },
```

Comment:
l: technically, this should be the default, so no need to set it

Reply:
Makes sense. Updated!